src/pages/Location.js

```diff
@@ -0,0 +1,111 @@
+// import React from "react";
```

Comment:
This commented code intended to be used later or can be removed?

Reply:
removed the old commented code!